data/remnant/remnant 2 cognizance.txt

```diff
@@ -1200,7 +1200,7 @@ mission "Remnant: Cognizance 25"
 				`	"No."`
 					decline
 	npc kill
-		government "Korath"
+		government "Kor Alaji"
```

Comment:
These are scouts testing a new prototype and exploring new space, not fanatics attacking the remnant.

```suggestion
		government "Korath"
```